jwt.go

```diff
@@ -19,13 +19,193 @@ var TimeFunc = time.Now
 // Header of the token (such as `kid`) to identify which key to use.
 type Keyfunc func(*Token) (interface{}, error)
 
+// For a type to be a Claims object, it must just have a Valid method that determines
+// if the token is invalid for any supported reason
+type Claimer interface {
```

Comment:
Alright, I updated the struct form to StandardClaims.

Renamed Claimer to Claims.

Do you mean you want to rename the interface method `Valid() error` -> `Claim() error`?